app/javascript/dashboard/routes/dashboard/settings/attributes/AddAttributes.vue

```diff
@@ -0,0 +1,194 @@
+                {{ model.option }}
+              </option>
+            </select>
+            <span v-if="$v.attributeModel.$error" class="message">
+              {{ $t('ATTRIBUTES_MGMT.ADD.FORM.MODEL.ERROR') }}
+            </span>
+          </label>
+        </div>
+        <div class="medium-12 columns">
+          <label :class="{ error: $v.attributeType.$error }">
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.TYPE.LABEL') }}
+            <select v-model="attributeType">
+              <option v-for="type in types" :key="type.id" :value="type.id">
+                {{ type.option }}
+              </option>
+            </select>
+            <span v-if="$v.attributeType.$error" class="message">
+              {{ $t('ATTRIBUTES_MGMT.ADD.FORM.TYPE.ERROR') }}
+            </span>
+          </label>
+        </div>
+        <div v-if="hasValue" class="medium-12 columns">
+          <label>
+            {{ $t('ATTRIBUTES_MGMT.ADD.FORM.KEY.LABEL') }}
+            <i class="ion-help" />
+          </label>
+          <p class="key-value text-truncate">
+            {{ convertToSlug }}
+          </p>
+        </div>
+        <div class="modal-footer">
+          <div class="medium-12 columns">
+            <woot-submit-button
+              :disabled="
+                $v.displayName.$invalid ||
+                  $v.description.$invalid ||
+                  uiFlags.isCreating
+              "
+              :button-text="$t('ATTRIBUTES_MGMT.ADD.SUBMIT')"
+            />
+            <button class="button clear" @click.prevent="onClose">
+              {{ $t('ATTRIBUTES_MGMT.ADD.CANCEL_BUTTON_TEXT') }}
+            </button>
+          </div>
+        </div>
+      </form>
+    </div>
+  </woot-modal>
+</template>
+
+<script>
+import { required, minLength } from 'vuelidate/lib/validators';
+import { mapGetters } from 'vuex';
+
+export default {
+  props: {
+    onClose: {
+      type: Function,
+      default: () => {},
+    },
+  },
+  data() {
+    return {
+      displayName: '',
+      description: '',
+      attributeModel: 0,
+      attributeType: 0,
+      models: [
+        {
+          id: 0,
+          option: 'Conversation',
+        },
+        {
+          id: 1,
+          option: 'Contact',
+        },
+      ],
+      types: [
+        {
+          id: 0,
+          option: 'Text',
+        },
+        {
+          id: 1,
+          option: 'Number',
+        },
+        {
+          id: 2,
+          option: 'Currency',
+        },
+        {
+          id: 3,
+          option: 'Percent',
+        },
+        {
+          id: 4,
+          option: 'Link',
+        },
+      ],
+      show: true,
+    };
+  },
+  computed: {
+    ...mapGetters({
+      uiFlags: 'getUIFlags',
+    }),
+    convertToSlug() {
+      return this.displayName
+        .toLowerCase()
+        .replace(/[^\w ]+/g, '')
+        .replace(/ +/g, '_');
+    },
+    hasValue() {
```

Comment:
I think this computed property is not needed. Just add a condition `displayName` is exist or not in the template